.cloudbuild/cloudbuild.yaml

```diff
@@ -14,7 +14,7 @@
 
 timeout: 7200s # 2 hours
 substitutions:
-  _SHARED_DEPENDENCIES_VERSION: '3.20.0' # {x-version-update:google-cloud-shared-dependencies:released}
+  _SHARED_DEPENDENCIES_VERSION: '3.21.0' # {x-version-update:google-cloud-shared-dependencies:released}
```

Comment:
Could you remind me why this one is `released` but the above two are `current`?

Reply:
The `cloudbuild.yaml` file is used for the Cloud Build trigger that actually publishes the docker images with every release whereas the `cloudbuild-test-*.yaml` files are mainly for verifying the docker images locally (therefore, they're also updated to SNAPSHOT). Since publishing the docker images for SNAPSHOT versions would be redundant, we only update `cloudbuild` when a release takes place.

Reply:
@burkedavison As discussed offline, modified the use of `released` to `current`. This will mitigate the risk of a manual trigger of the cloud build job accidentally overriding a published image.